lib/perl/Genome/Model/Tools/Vcf/EvaluateVcfs.pm

```diff
@@ -0,0 +1,374 @@
+package Genome::Model::Tools::Vcf::EvaluateVcfs;
+
+use strict;
+use warnings;
+
+use Cwd;
+use Path::Class;
+
+class Genome::Model::Tools::Vcf::EvaluateVcfs {
+    is => "Command::V2",
+    has_input => [
+        bedtools_version => {
```

Comment:
Unless I'm missing something (very possible!), neither this nor `joinx_version` nor `vcflib_version` are used in this command.